### PR TITLE
Add first RBAC wrapper resource s3 bucket

### PR DESCRIPTION
**Pull request by rifisdfds**
This PR adds an RBAC wrapped S3 bucket with some instructions for how to create future RBAC wrapped resources using it as an example

### CODE REVIEW

File: package/rbacwrappers/aws/bucket/definition.yaml [hunk 1]
```diff
@@ -0,0 +1,1447 @@
...
+                                  type: string
+                              type: object
+                          required:
+                          - redirect
+                          type: object
+                        type: array
+                    type: object
+                required:
+                - locationConstraint
+                type: object
```
Comment (samidbb) [inline package/rbacwrappers/aws/bucket/definition.yaml:1444]: this looks like a duplicate of line 27

File: package/rbacwrappers/aws/README.md [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+We should then add properties of the resources under the third `properties:` we defined above. We can copy and paste 
+these from the source code of the resource in aws-provider (using the desired release tag). For example, `https://raw.githubusercontent.com/crossplane/provider-aws/v0.20.1/package/crds/s3.aws.crossplane.io_buckets.yaml` and copy the forProvider.properties
+
+And make sure we end with making parameters required:
+
+```
+            required:
+            - parameters
+```
+
```
Comment (samidbb) [inline package/rbacwrappers/aws/README.md:69]: Adding something in line with "the complete definition.yaml will look like this" here, will increase readability and understanding of how it should work

**Comment by samidbb** [comment]
some comments on the naming conventions on the naming convention:
Composite resources:
xs3buckets.crossplane.dfds.cloud  (following the examples from Upbound)
vs
composites3buckets.crossplane.dfds.cloud

Claim:
S3Bucket
vs
S3BucketClaim

**Comment by samidbb** [comment]
also could the path be reduced to something like package/aws/s3bucket ?

File: package/rbacwrappers/aws/bucket/definition.yaml (hunk 1 above)
Comment (rifisdfds) [reply package/rbacwrappers/aws/bucket/definition.yaml]: Amended

File: package/rbacwrappers/aws/README.md (hunk 2 above)
Comment (rifisdfds) [reply package/rbacwrappers/aws/README.md:69]: Future changes to come

**Review by samidbb — APPROVED** [review]
will be dealing with suggestions in new branch

**Pull request merged**